pkg/resources/management.cattle.io/v3/setting/validator_test.go

```diff
@@ -63,75 +65,55 @@ func (t *retentionTest) toOldSetting() ([]byte, error) {
 	})
 }
 
+const iso8601Date = "20240108T000000Z"
+
+var rfc3339Date = time.Now().Truncate(time.Second).Format(time.RFC3339)
+
 var retentionTests = []retentionTest{
-	{
-		setting: "disable-inactive-user-after",
-		value:   "",
-		allowed: true,
-	},
-	{
-		setting: "delete-inactive-user-after",
-		value:   "",
-		allowed: true,
-	},
-	{
-		setting: "user-last-login-default",
-		value:   "",
-		allowed: true,
-	},
-	{
-		setting: "user-retention-cron",
-		value:   "",
-		allowed: true,
-	},
-	{
-		setting: "disable-inactive-user-after",
-		value:   "2h30m",
-		allowed: true,
-	},
-	{
-		setting: "delete-inactive-user-after",
-		value:   setting.MinDeleteInactiveUserAfter.String(),
-		allowed: true,
-	},
-	{
-		setting: "user-last-login-default",
-		value:   "2024-01-08T00:00:00Z",
-		allowed: true,
-	},
-	{
-		setting: "user-retention-cron",
-		value:   "* * * * *",
-		allowed: true,
-	},
-	{
-		setting: "disable-inactive-user-after",
-		value:   "1w",
-	},
-	{
-		setting: "delete-inactive-user-after",
-		value:   "2d",
-	},
-	{
-		setting: "user-last-login-default",
-		value:   "foo",
-	},
-	{
-		setting: "user-retention-cron",
-		value:   "* * * * * *",
-	},
-	{
-		setting: "disable-inactive-user-after",
-		value:   "-1h",
-	},
-	{
-		setting: "delete-inactive-user-after",
-		value:   "-1h",
-	},
-	{
-		setting: "delete-inactive-user-after",
-		value:   (setting.MinDeleteInactiveUserAfter - time.Second).String(),
-	},
+	// Empty values and defaults.
+	{"disable-inactive-user-after", "", "", true},
```

Comment:
Nit: Rather than repeating the tests, you could re-run the tests with something like `setting.Default = setting.Value` and `setting.Value = ""`. This would let you make sure that if a test case is covered for value it's covered for default too. Up to you though, I think there's an argument to be made that this is more clear so I don't think it absolutely needs to change.